cmd/gendbmdls/main.go

```diff
@@ -72,14 +74,17 @@ func main() {
 		Imports:         configureImports(),
 	})
 	if err != nil {
+		fmt.Println("error in setting up boilingcore command state")
```

Comment:
Do we need these println's?

Reply:
Although the `Println` itself is not required, but it should still be helpful in adding context to panic

Reply:
Yes makes sense. We can keep it.